builder/views.py

```diff
@@ -213,7 +214,7 @@ def update(request, draft):
 @require_http_methods(["POST"])
 @load_draft
 @require_permission
-def new_search(request, draft):
+def new_search(request: HttpRequest, draft: CodelistVersion):
```

Comment:
Although I'm generally a fan of type hits, AFAICT we're not using them anywhere else in this codebase so this driveby feels slightly out of place.

Reply:
Yeah fair enough. It was nice while I was working on it but probably should be part of a wider piece of work if we go down that route. I'll get rid.